### PR TITLE
Unbound: add WPAD

### PR DESCRIPTION
**Pull request by fabianfrz**
part of #2023

### CODE REVIEW

File: src/www/services_unbound.php [hunk 1]
```diff
@@ -267,6 +270,16 @@
                           </output>
                         </td>
                       </tr>
+                      <tr>
+                        <td><a id="help_for_enable_wpad" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("WPAD Records");?></td>
+                        <td>
+                          <input name="enable_wpad" type="checkbox" value="yes" <?=!empty($pconfig['enable_wpad']) ? "checked=\"checked\"" : "";?> />
+                          <output class="hidden" for="help_for_enable_wpad">
```
Comment (fichtner) [inline src/www/services_unbound.php]: we use a different style for help text now

**Comment by fichtner** [comment]
General question: is dnsmasq capable of this? or is this an unbound-only feature?

**Comment by fabianfrz** [comment]
I have not tried dnsmasq but as far as I know it can fake the A/AAAA record by adding the IP / Host to the hosts file.

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**